prototype/FFT_image_processing/image_processing.c

```diff
@@ -0,0 +1,414 @@
+#include <math.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <time.h>
+
+#define fft_real float
```

Comment:
I'm not sure to know how does this #define behave. You have previously defined fft_real in the header file and now you redefine it in image_processing.c. This looks like UB to me don't you agree ?

Reply:
There is nowhere else where `fft_real` is defined. `image_processing.c` has no header file, and the `1chipML` source is never imported.

Reply:
You're right, my bad.

> Symbol is defined from its definition until the end of the compilation unit